### PR TITLE
feat: revert to using normal faucet

### PR DESCRIPTION
**Pull request by kyranjamie**
> [Download the latest builds [4.0.0-beta.6]](https://github.com/blockstack/stacks-wallet/actions/runs/341854605).

That's the last time I use vscode built-in git. Checked out this pr blockstack/stacks-wallet#303 it the branch via git panel, had only a single commit with the entire tree in it, so a commit amend gone done broked stuff 🤔 🥴

This PR makes some changes to the CI task and adds some (very) simple error handling for faucet, and only sends a stacking faucet request when clicking the button while holding <kbd>alt</kbd>

### CODE REVIEW

File: app/components/home/balance-card.tsx [hunk 1]
```diff
@@ -55,18 +63,26 @@ export const BalanceCard: FC<BalanceCardProps> = props => {
...
+            size="md"
+            ml="tight"
+            isDisabled={requestingTestnetStx}
+            onClick={e => requestTestnetStacks(e)}
           >
-            🚰
-          </Box>
-          {requestingTestnetStx ? 'Requesting faucet' : 'Get testnet STX'}
-        </Button>
+            <Box
```
Comment (markmhx) [inline app/components/home/balance-card.tsx:74]: Maybe add a tooltip that provides guidance for alt-clicking?

File: configs/webpack.config.base.js [hunk 2]
```diff
@@ -64,6 +64,7 @@ export default {
   plugins: [
     new webpack.EnvironmentPlugin({
       NODE_ENV: 'production',
+      SHA: '',
```
Comment (markmhx) [inline configs/webpack.config.base.js:67]: This missing property value wouldn't be best caught downstream instead of setting to an empty string?

**Review by markmhx — APPROVED** [review]

File: app/components/home/balance-card.tsx (hunk 1 above)
Comment (kyranjamie) [reply app/components/home/balance-card.tsx:74]: But it's secret

File: configs/webpack.config.base.js (hunk 2 above)
Comment (kyranjamie) [reply configs/webpack.config.base.js:67]: This is just a default, empty is okay—good question though

File: app/components/home/balance-card.tsx (hunk 1 above)
Comment (markmhx) [reply app/components/home/balance-card.tsx:74]: I'm not sure it _has_ to be a secret. Community members may want to test Stacking as well, and I assume we can support several of them per day hitting this button?

Comment (kyranjamie) [reply app/components/home/balance-card.tsx:74]: Alright, will add a toolstip

Comment (kyranjamie) [reply app/components/home/balance-card.tsx:75]: See d585e6e

**Pull request merged**